<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>-60</y>
      <width>711</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signInButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logInButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QListWidget" name="friendConnectedList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>365</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="friendDisconnectedList">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>365</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="errorLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="friendRequestList">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>365</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>qrc:/Ressources/exit.png</normaloff>qrc:/Ressources/exit.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="callList">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>340</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <zorder>friendDisconnectedList</zorder>
   <zorder>friendConnectedList</zorder>
   <zorder>welcomeLabel</zorder>
   <zorder>usernameText</zorder>
   <zorder>passwordText</zorder>
   <zorder>signInButton</zorder>
   <zorder>logInButton</zorder>
   <zorder>errorLog</zorder>
   <zorder>friendRequestList</zorder>
   <zorder>disconnectButton</zorder>
   <zorder>callList</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServeur">
    <property name="title">
     <string>Serveur</string>
    </property>
    <addaction name="actionConnection"/>
    <addaction name="actionSearchForFriend"/>
   </widget>
   <addaction name="menuServeur"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnection">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionSearchForFriend">
   <property name="text">
    <string>SearchForFriend</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
